Strided domain

roshansamuel opened the pull request:
Fix for issue #158 by adding an empty constructor for `StridedDomain` in `blitz/array/domain.h`

If `strided-domain.cpp` within `testsuite/` compiles without error, the bug is fixed.

slayoo commented:
Thank you! Please do add the test to the testsuite list here: https://github.com/blitzpp/blitz/blob/master/testsuite/CMakeLists.txt
so it will be run on Travis and Appveyor CI runs

roshansamuel commented:
Hi! I have added the test to the testsuite list. I have also updated the strided-domain.cpp so that it sends an output to the terminal at the end of its execution.
